dialog/color_picker.go

```diff
@@ -186,7 +186,7 @@ func (p *colorAdvancedPicker) CreateRenderer() fyne.WidgetRenderer {
 					preview,
 				),
 			),
-			fyne.NewContainerWithLayout(layout.NewCenterLayout(), hex),
+			hex,
```

Comment:
I was unsure about this change, but when I looked at the golden images it made sense and looked good (I would not have been able to make this judgement from reading an XML string).

Perhaps the entry, and the colour channel entries, should become right aligned so they all match up?

Reply:
A separate PR to remove the center layout? I think that's going a bit far.
This change causes the entry to shrink further than it did before, so the field did not look good with the defined layout.

Reply:
I didn't request a separate PR to remove the center layout, I had wondered about it, and guessed that either approach would break the test image. As I said, this change made sense and looked good, so it can stay. The comment was more thinking out aloud, sorry if that wasn't clear.